src/session.rs

```diff
@@ -563,136 +708,17 @@ where
     /// [the spec]: https://www.w3.org/TR/webdriver/#list-of-endpoints
     fn issue_wd_cmd(
         &mut self,
-        cmd: WebDriverCommand<webdriver::command::VoidWebDriverExtensionCommand>,
+        cmd: Box<dyn WebDriverCompatibleCommand + Send>,
```

Comment:
This can definitely be generic. No need to force the boxing here.

Reply:
Hmm, are you sure?

It is called from session:rs:492
```rust
                    Cmd::WebDriver(request) => {
                        // looks like the client setup is falling back to legacy params
                        // keep track of that for later
                        if request.is_legacy() {
                            self.is_legacy = true;
                        }

                        self.ongoing = Ongoing::WebDriver {
                            ack,
                            fut: Box::pin(self.issue_wd_cmd(request)),
                        };
                    }
```

the type of `request` here is `Box<dyn WebDriverCompatibleCommand + Send>`

Reply:
Yeah, but then `impl WebDriverCompatibleCommand + Send + 'static` would work.

Reply:
I'm having trouble getting this to work. I'm getting this:
```
error[E0277]: the size for values of type `dyn WebDriverCompatibleCommand + Send` cannot be known at compilation time
   --> src\session.rs:501:61
    |
501 | ...                   fut: Box::pin(self.issue_wd_cmd(request)),
    |                                          ------------ ^^^^^^^ doesn't have a size known at compile-time
    |                                          |
    |                                          required by a bound introduced by this call
    |
    = help: the trait `Sized` is not implemented for `dyn WebDriverCompatibleCommand + Send`
    = note: required because of the requirements on the impl of `WebDriverCompatibleCommand` for `Box<dyn WebDriverCompatibleCommand + Send>`
note: required by a bound in `session::Session::<C>::issue_wd_cmd`
   --> src\session.rs:711:19
    |
709 |     fn issue_wd_cmd(
    |        ------------ required by a bound in this
710 |         &self,
711 |         cmd: impl WebDriverCompatibleCommand + Send + 'static,
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `session::Session::<C>::issue_wd_cmd`
```